bokehjs/src/lib/models/glyphs/webgl/regl_wrap.ts

```diff
@@ -84,6 +88,18 @@ export class ReglWrapper {
     return func
   }
 
+  public rect_no_hatch(): ReglRenderFunction {
+    if (this._rect_no_hatch === undefined)
```

Comment:
Unless you're distinguishing `null` from `undefined`, we prefer `== null` (as in idiom).

Reply:
I had used this in quite a few places; I've changed them all to `== null` now.